src/test/scala/com/microsoft/hyperspace/index/E2EHyperspaceRulesTests.scala

```diff
@@ -91,6 +91,18 @@ class E2EHyperspaceRulesTests extends HyperspaceSuite {
     verifyIndexUsage(query, Seq(getIndexFilesPath(indexConfig.indexName)))
   }
 
+  test("E2E test for case insensitive filter query utilizing indexes.") {
```

Comment:
Can you add a test where changing the sensitivity config changes the behavior?

Reply:
thanks @imback82 , added